src/containers/months-list.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Month from './month';
+
+class MonthsList extends Component {
+
+  renderList() {
+    return this.props.months.map( (month) => {
+      return (
+        <Month key={month.abbr} abbr={month.abbr} />
+      );
+    });
+  };
```

Comment:
This can be simplified to:

    renderList() {
        return this.props.months.map(month => (
            <Month key={month.abbr} abbr={month.abbr} />
        ));
    };

or

    renderList() {
        return this.props.months.map(month => 
            <Month key={month.abbr} abbr={month.abbr} />
        );
    };

(Basically, whatever the shortest version is that still compiles in JSX; I don't know if it gets funny about HTML tags in lambdas.)